### PR TITLE
Update v2024.2.0 deps

### LINKED ISSUE #649: Update infrastructure dependencies for v2024.2
Update any dependencies other than Frontend and Backend that need to be updated:

- [x] Enketo
- [x] pyxform-http
- [x] Node
  - Web Forms requires Node 20.13.1+, but Central currently uses 20.12.2 in production and 20.10.0 locally.
  - `service`
  - `nginx`
  - `secrets`

### PR DESCRIPTION
**Pull request by lognaturel**
Closes #649

#### What has been done to verify that this works as intended?
Tried on dev, verified basic happy paths.

#### Why is this the best possible solution? Were any other approaches considered?
I searched for the previous node version so hopefully got all of those this time!

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
There are no significant expected changes in behavior. pyxform and Enketo releases are relatively small. No major version bumps anywhere.

#### Does this change require updates to documentation? If so, please file an issue [here](https://github.com/getodk/docs/issues/new) and include the link below.
No.

#### Before submitting this PR, please make sure you have:

- [x] branched off and targeted the `next` branch OR only changed documentation/infrastructure (`master` is stable and used in production)
- [x] verified that any code or assets from external sources are properly credited in comments or that everything is internally sourced

### CODE REVIEW

File: nginx.dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ RUN files/prebuild/build-frontend.sh
 
 # when upgrading, look for upstream changes to redirector.conf
 # also, confirm setup-odk.sh strips out HTTP-01 ACME challenge location
-FROM jonasal/nginx-certbot:5.0.1
+FROM jonasal/nginx-certbot:5.0.4
```
Comment (matthew-white) [inline nginx.dockerfile:17]: CircleCI seems to be complaining about this image for some reason.

**Review by matthew-white — APPROVED** [review]

File: nginx.dockerfile (hunk 1 above)
Comment (lognaturel) [reply nginx.dockerfile]: I switched some numbers around is why. Sorry, I intended to open as draft!

Comment (matthew-white) [reply nginx.dockerfile]: Sorry, didn't mean to arrive early to the party!

Comment (lognaturel) [reply nginx.dockerfile]: Haha, I've done my checks and gotten my numbers straight now so I'll take your approval. Thank you! 🥳

**Comment by matthew-white** [comment]
Do you have time to update Node in central-backend and central-frontend as well (package.json / Volta / maybe CI)? npm yells at me whenever I install dependencies for Frontend because Web Forms is expecting a later Node. I'm happy to do that update also.

**Comment by lognaturel** [comment]
Yep, can do that now.

**Pull request merged**